packages/synapse-react-client/src/utils/functions/RegularExpressions.ts

```diff
@@ -5,6 +5,14 @@ import { normalizeSynPrefix } from './EntityTypeUtils'
 // note - had to add an escape character for the second and third forward slash in the regex above
 export const DOI_REGEX = /^10.\d{4,9}\/[-._;()/:a-z0-9]+$/i
 
+export function convertDoiToLink(doi: string) {
```

Comment:
Can you just add a couple of tests for this, at least these input cases, to ensure future changes to this method do not break the components using it?

* Valid DOI
* Malformed DOI
* Empty string